job-dsls/jobs/prod/prod_kogito_create_images_templates_and_sources.groovy

```diff
@@ -0,0 +1,31 @@
+/**
+* Create the OpenShift images templates and sources
+*/
+def scriptTemplate = this.getClass().getResource("job-scripts/prod_kogito_create_images_templates_and_sources.jenkinsfile").text
+def parsedScript = scriptTemplate.replaceAll(/<%=\s*(\w+)\s*%>/) { config[it[1]] ?: '' }
+
+def folderPath = "PROD"
+folder(folderPath)
+
+pipelineJob("${folderPath}/kogito-create-container-sources") {
```

Comment:
the rhba job is called **create-images-templates-and-sources**, what if we call this one **kogito-create-images-sources** to keep somehow aligned.